### ISSUE TITLE
Form elements are missing labels

### ISSUE THREAD

**Issue by sarahocarroll** [socarra/automation_assessment#5]
- Navigate to https://automationintesting.online/
- Run the axe Dev tools - full page scan
- This reports that a number of form elements do not have a label

<img width="2457" height="1202" alt="Image" src="https://github.com/user-attachments/assets/63ce18d9-f964-4803-bd4a-1d0bbc99f160" />
